src/main/java/com/artipie/rpm/pkg/HeaderTags.java

```diff
@@ -329,6 +329,14 @@ public String sourceRmp() {
         return this.meta.header(Header.HeaderTag.FILEMODES).asInts();
     }
 
+    /**
+     * Get the file flags header.
+     * @return Value of header tag FILEFLAGS.
+     */
+    public int[] fileFlags() {
+        return this.meta.header(Header.HeaderTag.FILEFLAGS).asInts();
```

Comment:
I'd suggest add separate test for this new method into `HeaderTagsTest.java`

Reply:
Oops, I forgot it :(